### PR TITLE
Ford-Fulkerson in Java and Python implemented

### PR DESCRIPTION
**Pull request by Riyazul555**
Closes issues #22  and #21  both in same PR

### CODE REVIEW

File: Ford-Fulkerson Algorithm/FordFulkerson.py [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+        for i in G[nod]:
+            if not dad[i] and C[nod][i] > 0:
+                dad[i] = nod
+                Q.append(i)
+    return dad[n]
+
+if __name__ == "__main__":
+    with open(sys.argv[1], 'r') as fin, open(sys.argv[2], 'w') as fout:
+        n, m = map(int, fin.readline().split())
+        G = [[] for _ in range(n + 5)]
```
Comment (iuliagroza) [inline Ford-Fulkerson Algorithm/FordFulkerson.py:23]: Why do we allocate extra "+5" size for the adjacency list of the graph?

File: Ford-Fulkerson Algorithm/FordFulkerson.py [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    while Q:
+        nod = Q.popleft()
+        for i in G[nod]:
+            if not dad[i] and C[nod][i] > 0:
+                dad[i] = nod
+                Q.append(i)
+    return dad[n]
+
+if __name__ == "__main__":
+    with open(sys.argv[1], 'r') as fin, open(sys.argv[2], 'w') as fout:
```
Comment (iuliagroza) [inline Ford-Fulkerson Algorithm/FordFulkerson.py:21]: Might be wrong, but shouldn't `sys` be imported for using `sys.argv`?

File: Ford-Fulkerson Algorithm/FordFulkerson.py
Comment (iuliagroza) [inline Ford-Fulkerson Algorithm/FordFulkerson.py:1]: Rename to "Ford-Fulkerson/ford-fulkerson.py" for snake_case.

File: Ford-Fulkerson Algorithm/FordFulkerson.java
Comment (iuliagroza) [inline Ford-Fulkerson Algorithm/FordFulkerson.java:1]: Rename to "Ford-Fulkerson/FordFulkerson.java".

**Review by iuliagroza — COMMENTED** [review]
For this PR, we can leave it like that, but for the future, I think dedicating a PR to a single issue instead of multiple is much easier and more clear.

**Comment by Riyazul555** [comment]
Ok 
No issues
I am closing this PR and will make a new one

**Pull request closed without merging**